ts/torch_handler/distributed/base_neuronx_continuous_batching_handler.py

```diff
@@ -0,0 +1,411 @@
+import logging
+import os
+
+import torch
+import torch_neuronx
+from transformers import AutoModelForCausalLM, AutoTokenizer
+from transformers_neuronx.config import ContinuousBatchingConfig, NeuronConfig
+from transformers_neuronx.module import save_pretrained_split
+from transformers_neuronx.sampling import select_tokens
+
+from ts.context import Context
+from ts.handler_utils.utils import import_class
+from ts.torch_handler.base_handler import BaseHandler
+
+logger = logging.getLogger(__name__)
+
+
+class BaseNeuronXContinuousBatchingHandler(BaseHandler):
```

Comment:
Better to at least write an e2e test for this handler. The e2e test should queue multiple requests and check for results to make sure ids are not mixed up. Better to test methods as well.

Reply:
test_llm_streaming_response.py is a tool for manual e2e testing, which is a workaround solution for the inf2 environment dependency.